### PR TITLE
Add logging for commands and Update the help message

### LINKED ISSUE #47: Add logging to Commands
DeleteCommand and ListCommand: https://github.com/AY1920S2-CS2113-T14-1/tp/issues/60 Add logging to DeleteCommand and ListCommand

### LINKED ISSUE #69: Update Help Message

### PR DESCRIPTION
**Pull request by sliu107**
Some commands require further information from users, so I am not sure whether the logging information I added in those classes are necessary or not.

And for bye command, I left  logging information just to make a seperation between each run in the logging file. I wonder whether rewritting the log files or creating different ones for each run is a better solution.

### CODE REVIEW

File: src/main/java/seedu/tp/commands/GroupCommand.java [hunk 1]
```diff
@@ -24,7 +24,9 @@ public GroupCommand(GroupFactory groupFactory, GroupList groupList) {
 
     @Override
     public void execute() throws InvalidFlashcardIndexException {
+        LOGGER.info("Creating a new group");
```
Comment (Zhilin-Huang) [inline src/main/java/seedu/tp/commands/GroupCommand.java]: Consider adding `...` to the end of the string to be consistent with other log messages

File: src/main/java/seedu/tp/commands/OtherFlashcardCommand.java [hunk 2]
```diff
@@ -36,7 +36,9 @@ public FlashcardList getFlashcardList() {
 
     @Override
     public void execute() throws UnrecognizedFlashcardTypeException {
+        LOGGER.info("Creating an other flashcard and adding it to the flashcard list");
```
Comment (Zhilin-Huang) [inline src/main/java/seedu/tp/commands/OtherFlashcardCommand.java]: Same here

File: src/main/java/seedu/tp/commands/PriorityCommand.java [hunk 3]
```diff
@@ -51,9 +51,12 @@ public int getIndex() {
     @Override
     public void execute() throws InvalidFlashcardIndexException {
         try {
+            LOGGER.info("Setting the priority for the flashcard " + index);
```
Comment (Zhilin-Huang) [inline src/main/java/seedu/tp/commands/PriorityCommand.java]: Same here

File: src/main/java/seedu/tp/commands/ReviewedCommand.java [hunk 4]
```diff
@@ -38,10 +38,13 @@ public int getIndex() {
     @Override
     public void execute() throws InvalidFlashcardIndexException {
         try {
+            LOGGER.info("Setting flashcard " + index + " as reviewed");
```
Comment (Zhilin-Huang) [inline src/main/java/seedu/tp/commands/ReviewedCommand.java]: Same here

File: src/main/java/seedu/tp/commands/ShowCommand.java [hunk 5]
```diff
@@ -39,9 +39,12 @@ public int getIndex() {
     @Override
     public void execute() throws InvalidFlashcardIndexException {
         try {
+            LOGGER.info("Showing the information of flashcard " + index);
```
Comment (Zhilin-Huang) [inline src/main/java/seedu/tp/commands/ShowCommand.java]: Same here

File: src/main/java/seedu/tp/commands/TimelineCommand.java [hunk 6]
```diff
@@ -14,6 +14,8 @@ public TimelineCommand(FlashcardList flashcardList, Ui ui) {
 
     @Override
     public void execute() {
+        LOGGER.info("Listing the flashcards in time order");
```
Comment (Zhilin-Huang) [inline src/main/java/seedu/tp/commands/TimelineCommand.java]: Same here

**Review by Zhilin-Huang — COMMENTED** [review]
The logging generally looks good to me, except for some places that could be improved for consistency

**Review by Zhilin-Huang — APPROVED** [review]

**Review by quinnyyy — APPROVED** [review]

**Pull request merged**